kiwi_import

```
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+require 'fileutils'
+require 'optparse'
+
+options = {}
+OptionParser.new do |opts|
+  opts.on("--outdir THE_PATH", "the output directory") do |outdir|
+    options[:outdir] = outdir
+  end
+end.parse!
+
+def get_kiwi_archive(path)
+  kiwi_archive = Dir[File.join(path, '/*.kiwi.txz')]
+  if kiwi_archive.length > 1
+    abort('More than one kiwi archives found')
+  elsif kiwi_archive.length == 0
+    abort('No kiwi archive found')
+  end
+
+  kiwi_archive[0]
+end
+
+def extract_archive(archive, outdir)
+  cmd = "tar Jxf #{archive} -C #{outdir}"
+  system(cmd)
+end
+
+def create_root_archive(outdir)
+  if File.directory?(outdir)
```

If outdir not existing, errors will be silently dropped. Better raise exception if no outdir given or outdir does not exist